spec/views/idv/by_mail/letter_enqueued/show.html.erb_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe 'idv/by_mail/letter_enqueued/show.html.erb' do
-  let(:service_provider) { '🔒🌐💻' }
+  let(:service_provider) { create(:service_provider) }
```

Comment:
AFAICT we were always expecting an instance of a ServiceProvider, not a (emoji-filled) string, so I'm not sure how this ever passed.

Reply:
Yeah that's not ideal. It looks like this service_provider gets passed into the Idv::Session constructor which then gets read by the presenter in a basic `if sp` type thing. So any truthy value would pass here